### ISSUE TITLE
'Restart Frigate' did not restart the 'Frigate NVR Home Assistant Add-on'

### ISSUE THREAD

**Issue by wire67** [blakeblackshear/frigate#3880]
### Describe the problem you are having

'Restart Frigate' did not restart the 'Frigate NVR Home Assistant Add-on'

there was no error about not being able to restart, but I was able to see from the log '/hassio/addon/ccab4aaf_frigate/logs' that it did not restart despite many attempts to press the 'Restart Frigate' button in the top right corner menu of '/ccab4aaf_frigate/dashboard'. 

Pressing the 'Restart' button from the addon page '/hassio/addon/ccab4aaf_frigate/info' worked immediately.

### Version

0.10.1-83481AF

### Frigate config file

```yaml
# yaml-language-server: $schema=http://ccab4aaf-frigate:5000/api/config/schema

mqtt:
  host: 'homeassistant.local'
  user: mqtt
  password: ''
  port: 1883
  
record:
  enabled: false # TAKES TOO MUCH HARDDRIVE
  events:
    retain:
      default: 2
  retain_days: 2
  
objects:
  # Optional: list of objects to track from labelmap.txt (default: shown below)
  track:
    - person
    
detect:
  # Optional: desired fps for your camera for the input with the detect role (default: shown below)
  # NOTE: Recommended value of 5. Ideally, try and reduce your FPS on the camera.
  fps: 1
    
    
# Optional: Configuration for the jpg snapshots written to the clips directory for each event
# NOTE: Can be overridden at the camera level
snapshots:
  # Optional: Enable writing jpg snapshot to /media/frigate/clips (default: shown below)
  # This value can be set via MQTT and will be updated in startup based on retained value
  enabled: true
  # Optional: Camera override for retention settings (default: global values)
  retain:
    # Required: Default retention days (default: shown below)
    default: 2

cameras:
  frontcam:
    ffmpeg:
      input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport udp -stimeout 5000000 -use_wallclock_as_timestamps 1
      inputs:
        # (1920x1080 5-15 fps)
        - path: rtsp://admin:PWD@192.168.8.16:554/onvif1 # 1~2
          roles:
            - record
        # (320x240 5-15 fps)
        - path: rtsp://admin:PWD@192.168.8.16:554/onvif2 # 1~2
          roles:
            - detect
            - rtmp
    detect:
      width: 320
      height: 240
  storagecam:
    ffmpeg:
      inputs:
        - path: rtsp://admin:PWD@192.168.8.101/live/ch00_0 # 0~1
          roles:
            - record
        - path: rtsp://admin:PWD@192.168.8.101/live/ch00_0 # 0~1
          roles:
            - detect
            - rtmp
    detect:
      width: 640
      height: 360
  backcam:
    ffmpeg:
      inputs:
        - path: rtsp://admin:PWD@192.168.8.102/live/ch00_0 # 0~1
          roles:
            - record
        - path: rtsp://admin:PWD@192.168.8.102/live/ch00_0 # 0~1
          roles:
            - detect
            - rtmp
    detect:
      width: 640
      height: 360
      
  espcam1:
    ffmpeg:
      inputs:
        - path: http://192.168.8.20:8080
          roles:
            - detect
          input_args:
            - -avoid_negative_ts
            - make_zero
            - -fflags
            - nobuffer
            - -flags
            - low_delay
            - -strict
            - experimental
            - -fflags
            - +genpts+discardcorrupt
            - -use_wallclock_as_timestamps
            - "1"
      output_args:
        detect: -f rawvideo -pix_fmt yuv420p
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v libx264 -an
        rtmp: -c:v libx264 -an -f flv
    detect:
      width: 640
      height: 480
      # fps: 25
    objects:
      track:
        - person
    snapshots:
      enabled: True
      timestamp: True
      bounding_box: False
```

### Relevant log output

```shell
There was nothing about the restart failing in log, but I did see the following error which is why I needed to restart in the first place (this is another issue, Frigate was unable to connect to MQTT until it had restart):

frigate.mqtt ERROR : Unable to connect to MQTT server: MQTT Not authorized
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Other

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wireless

### Camera make and model

yoosee, v380

**Comment by wire67** [comment]
Still an issue with v0.11, restart button doesn't work at all in any situation.

**Comment by wire67** [comment]
Up

**Comment by NickM-27** [comment]
This should be fixed in 0.12
